x-pack/plugins/actions/server/sub_action_framework/case.ts

```diff
@@ -79,7 +79,7 @@ export abstract class CaseConnector<Config, Secrets>
     incidentId: string;
     incident: Record<string, unknown>;
   }): Promise<ExternalServiceIncidentResponse>;
-  public abstract getIncident({ id }: { id: string }): Promise<ExternalServiceIncidentResponse>;
+  public abstract getIncident({ id }: { id: string }): Promise<Record<string, unknown>>;
```

Comment:
Get incident should return all attributes of incident. As it is also used in create and update incident.